hopsanc/hopsanc.pro

```diff
@@ -0,0 +1,54 @@
+# -------------------------------------------------
+# Global project options
```

Comment:
I would also like a CMake file, please :)
Or I can make one if you want

Reply:
I created it, but I am no CMake export so you might want to take a look at it.

Reply:
looks good but did not test, you also need to include in in the top-level cmake file, "add_subdirectory(hopsanc)
"
Also I noticed it is added as a static library, but in the qt .pro file it is a shared library. Do know what is best in this case, but I guess it should be shared, so that Julia can load it at runtime ?